### DISCUSSION TITLE
Random tiny utility: covid.params.json to csv with pre-defined/fixed column headers

### DISCUSSION THREAD

**Discussion by sairaghavak** [neherlab/covid19_scenarios#530] [General]
Hi

Developed a tiny util to convert covid.params.json to a pre-defined csv format.
Source code here: https://github.com/sairaghavak/covid19-simulation-params-json-to-csv
Web UI here: http://sairaghavak.herokuapp.com/covid19params/
REST endpoint: http://sairaghavak.herokuapp.com/covid19params/api/jsontocsv

Thanks

**Comment by nnoll** [comment]
Thanks for your contribution!
